translations/kubectl/zh_TW/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,89 @@
+# Test translations for unit tests.
+# Copyright (C) 2017
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR warmchang@outlook.com, 2017.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: hello-world\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-05-26 08:28+0800\n"
+"PO-Revision-Date: 2017-06-01 10:51+0800\n"
+"Last-Translator: William Chang <warmchang@outlook.com>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.0.2\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"Language-Team: \n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Language: zh\n"
+
+#: pkg/kubectl/cmd/apply.go:104
+msgid "Apply a configuration to a resource by filename or stdin"
+msgstr "通過檔案名或標準輸入流(stdin)對資源進行配置"
+
+#: pkg/kubectl/cmd/config/delete_cluster.go:39
+msgid "Delete the specified cluster from the kubeconfig"
+msgstr "刪除 kubeconfig 檔案中指定的叢集(cluster)"
+
+#: pkg/kubectl/cmd/config/delete_context.go:39
+msgid "Delete the specified context from the kubeconfig"
+msgstr "刪除 kubeconfig 檔案中指定的context"
```

Comment:
Optional: Add a space between Chinese & English character so that it looks consistent